cmake/modules/RootMacros.cmake

```diff
@@ -1157,7 +1157,7 @@ function(ROOT_INSTALL_HEADERS)
       set (dst ${CMAKE_BINARY_DIR}/include/${include_file})
       add_custom_command(
         OUTPUT ${dst}
-        COMMAND ${CMAKE_COMMAND} -E copy ${src} ${dst}
+        COMMAND ${CMAKE_COMMAND} -E create_symlink ${src} ${dst}
```

Comment:
I could pull this change out, but without it I cannot efficiently jump between compiler output and header files in Visual Studio. I do not know if this has any side effects.

Reply:
This is a noticeable change and unrelated to the PR's title.   It may or may not be the right solution to the problem (which deserves a solution :)).  Could you open a separate PR for it?  Thanks.

Reply:
Sure! It's #6010 .